_components/pages/Docs.tsx

```diff
@@ -205,7 +205,7 @@ export default function Docs() {
               <div className="dg-lg-block mb-5">
                 <a
                   className="btn btn-primary btn-arrow get-started-button"
-                  href="https://xrpl.org/xrpl-quickstart.html"
+                  href="./tutorials/quickstart/index.md"
                 >
                   Get Started
                 </a>
```

Comment:
The change here as written won't work because raw HTML `<a>` tags aren't smart enough to know that a link to a `md` file should go to the page built from that file. However, if you use redocly's `<Link>` element instead, it can work.

```suggestion
                <Link
                  className="btn btn-primary btn-arrow get-started-button"
                  to="./tutorials/quickstart/index.md"
                >Get Started</Link>
```

(Don't forget to change `href` to `to` _and_ change the closing tag.)